subsys/mgmt/ec_host_cmd/backends/ec_host_cmd_backend_spi_stm32.c

```diff
@@ -195,8 +197,7 @@ static int prepare_rx(struct ec_host_cmd_spi_ctx *hc_spi);
 			.dma_callback = dma_callback,                                              \
 			.block_count = 2,                                                          \
 	},                                                                                         \
-	.fifo_threshold =                                                                          \
-		STM32_DMA_FEATURES_FIFO_THRESHOLD(DT_DMAS_CELL_BY_NAME(id, dir, features)),
+	.fifo_threshold = STM32_DMA_FEATURES_FIFO_THRESHOLD(STM32_DMA_FEATURES_ID(id, dir)),
```

Comment:
The new macro is nice, but what does it bring compare to existing definition ?
```
#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32_dma_v1)
#define STM32_DMA_FEATURES_FIFO_THRESHOLD(features)	(features & 0x3)
#else
#define STM32_DMA_FEATURES_FIFO_THRESHOLD(features)	0
#endif
```

Reply:
Ok, got it.
Actually those (`STM32_DMA_FEATURES`) should be `_INST_` variants and we should have provided "non compat implicit" versions as you need. But you're proposal is fine as well indeed